manifests/init.pp

```diff
@@ -261,6 +264,7 @@
   Optional[Integer]                                   $udev_event_timeout = undef,
   Optional[Enum['early', 'late', 'never']]            $udev_resolve_names = undef,
   Optional[Variant[Integer,String]]                   $udev_timeout_signal = undef,
+  Boolean                                             $udev_reload = true,
```

Comment:
For backwards compatibility, the default should probably be `false`

Reply:
Sure